lib/algfld.gi

```diff
@@ -1279,9 +1279,9 @@ local opt,irrfacs, coeffring, i, factors, ind, coeffs, val,
       Add( factors, factor );
       q:= Quotient( R, g, factor );
       while q <> fail do
-	Add( factors, factor );
-	g:= q;
-	q:= Quotient( R, g, factor );
+    Add( factors, factor );
+    g:= q;
+    q:= Quotient( R, g, factor );
```

Comment:
Indentation is broken here. I would advise against trying to remove tabs with a fully automated script: in many places our tabs are 8 wide, but I think there are exceptions with 4 wide tabs (and who knows what else)